tests/yadism/cf/test_nc_asy_polarized.py

```diff
@@ -1,130 +1,129 @@
 import numpy as np
 from utils import MockESF
 
-from yadism.coefficient_functions.fonll import g1_nc as f_g1_nc
-from yadism.coefficient_functions.fonll import g1_nc_raw
+from yadism.coefficient_functions.asy import g1_nc as f_g1_nc
+from yadism.coefficient_functions.asy import g1_nc_raw
 from yadism.coefficient_functions.heavy import g1_nc as h_g1_nc
 
+# class Test_g1_raw:
```

Comment:
This test should not be removed, it tests the `g1` asymptotic.

Reply:
That's why I uncommitted it instead of deleting it, I just didn't want the spam from the failing tests